File: Offer2/0021/main.py
from typing import List


class Solution:
    def exchange(self, nums: List[int]) -> List[int]:
        array1: List[int] = []
        array2: List[int] = []
        for value in nums:
            if int(value % 2) != 0:
                array1.append(value)
            else:
                array2.append(value)
        array1.extend(array2)
        return array1
        

File: Offer2/0021/main.go
package main

func exchange(nums []int) []int {
	var array1, array2 []int
	for _, value := range nums {
		if value%2 != 0 {
			array1 = append(array1, value)
		} else {
			array2 = append(array2, value)
		}
	}
	return append(array1, array2...)
}
